Example/PaymentSheet Example/PaymentSheet Example/ExampleEmbeddedElementCheckoutViewController.swift

```diff
@@ -91,6 +91,7 @@ class ExampleEmbeddedElementCheckoutViewController: UIViewController {
     @objc
     func didTapPaymentMethodButton() {
         let paymentMethodsViewController = PaymentMethodsViewController(embeddedPaymentElement: embeddedPaymentElement, needsDismissal: { [weak self] in
+            self?.embeddedPaymentElement.presentingViewController = self
```

Comment:
I'm not sure why this is needed -- it seems to be set in the init of PyamentMethodsViewController

Reply:
That is because the presenting view controller is set when the PaymentMethodsViewController is presented, so when it dismisses we need to set the presentingViewController back to self.

https://github.com/stripe/stripe-ios/blob/master/Example/PaymentSheet%20Example/PaymentSheet%20Example/ExampleEmbeddedElementCheckoutViewController.swift#L365

Reply:
chatted offline -- this is needed due to authContext on confirm.